src/rules.js

```diff
@@ -1903,6 +1903,80 @@ module.exports = [
     },
     /**
      ==================================================================
+     TRANSITION
+     ==================================================================
+     */
+    {
+        type: 'pattern',
+        id: 'transition',
+        name: 'Transition',
+        prefix: '.Trs-',
+        properties: ['transition'],
+        allowCustom: true,
+        allowCustomAutoSuffix: false,
+        allowSuffixToValue: true
```

Comment:
I think because it makes the syntax less clear.  We'd have to support this for all rules that have `allowSuffixToValue`, and are we sure that won't introduce ambiguity or conflict in other styles' values?  For that reason, I think that if we want to go down this path, we need to carefully consider and test this change.

At any rate, for now this syntax doesn't work in Atomizer, so for the sake of your pull request, it should be disabled in  the rules file.  We can file an issue for the feature support and enable this flag later if support is added.